src/site/content/en/learn/design/responsive-images/index.md

```diff
@@ -164,13 +165,44 @@ For a hero image above the fold, use a `loading` value of `eager`.
 >
 ```
 
-For an important image you can tell the browser to pre-fetch the image in the head of your document.
+### Priority hints
+
+For important images-such as the (LCP)[/lcp/] image, you can further prioritise the loading using [Priority Hints](/priority-hints/) by setting the `fetchpriority` attribute to `high`:
+
+```html/5-6
+<img
+ src="hero.jpg"
+ alt="A description of the image."
+ width="1200"
+ height="800"
+ loading="eager"
+ fetchpriority="high"
+>
+```
+
+This will tell the browser to fetch the image right away, and at high priority, rather than waiting until the browsder has completed layout when images are normally fetched.
```

Comment:
```suggestion
This will tell the browser to fetch the image right away, and at high priority, rather than waiting until the browser has completed layout when images are normally fetched.
```